mlflow/gateway/config.py

```diff
@@ -46,6 +46,7 @@ class Provider(str, Enum):
     # Note: The following providers are only supported on Databricks
     DATABRICKS_MODEL_SERVING = "databricks-model-serving"
     DATABRICKS = "databricks"
+    CLARIFAI = "clarifai"
```

Comment:
Can we move this right below "BEDROCK" (since the note on line 46 doesn't apply to CLARIFAI)?